### PR TITLE
slightly populated database

### PR DESCRIPTION
**Pull request by SJouzu**

### CODE REVIEW

File: db/tup_e-learning.sql [hunk 1]
```diff
@@ -115,6 +163,17 @@ CREATE TABLE `students` (
   `E-mail` varchar(30) NOT NULL
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+--
+-- Dumping data for table `students`
+--
+
+INSERT INTO `students` (`Student_ID`, `L_name`, `F_name`, `M_name`, `Course`, `Section`, `E-mail`) VALUES
```
Comment (henouji) [inline db/tup_e-learning.sql:170]: Section is just "A" or "B"

File: db/tup_e-learning.sql [hunk 2]
```diff
@@ -85,6 +122,17 @@ CREATE TABLE `professors` (
   `E-mail` varchar(30) NOT NULL
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+--
+-- Dumping data for table `professors`
+--
+
```
Comment (henouji) [inline db/tup_e-learning.sql:128]: Add middle initial in the next updates

**Review by henouji — APPROVED** [review]
Change sections please, tho merge for the meantime

File: db/tup_e-learning.sql (hunk 1 above)
Comment (SJouzu) [reply db/tup_e-learning.sql:170]: ok

**Pull request merged**